test/runtests.jl

```diff
@@ -659,4 +659,33 @@ end
     @test RGB(Gray(0.2), Gray(0.3), Gray(0.4)) == RGB(0.2, 0.3, 0.4)
 end
 
+@testset "RGB565" begin
+    c = RGB565(31, 63, 31)  # white
+    @test blue(c) == 31
```

Comment:
```suggestion
    @test blue(c) === N3f5(1)
```

Similar changes throughout below.